CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+- Do not add double BT/BLE items
+- New version of libdivecomputer 
+- Fixed unstability crashes in reverse geo lookup fucntion (dive site edit)
```

Comment:
is that "unstability" or "instability" ?
I think the former isn't an English word